Sources/SwiftToolchain/ToolchainManagement.swift

```diff
@@ -78,7 +78,8 @@ extension FileSystem {
     if let versionSpec = versionSpec {
       if let url = URL(string: versionSpec),
         let filename = url.pathComponents.last,
-        let match = versionRegEx.matchGroups(in: filename).first?.first {
+        let match = versionRegEx.matchGroups(in: filename).first?.first
+      {
```

Comment:
<!--
  1 failure:  Opening braces sh...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/SwiftToolchain/ToolchainManagement.swift;
  Line: 82;
-->
- :no_entry_sign: Opening braces should be preceded by a single space and on the same line as the declaration. (`opening_brace`)